Code/Tools/ProjectManager/Source/GemCatalog/GemCatalogScreen.cpp

```diff
@@ -730,4 +682,14 @@ namespace O3DE::ProjectManager
     {
         return ProjectManagerScreen::GemCatalog;
     }
+
+    QString GemCatalogScreen::GetTabText()
+    {
+        return tr("Gems");
```

Comment:
So what would we translate "Gems" to in another language?

Reply:
Someone a long time ago said it was actually an acronym for "Game Engine Module" or "Modification" or something similar to that.  I agree though, it's more of a feature name than a translatable term.

Reply:
I thought it was based off of [Ruby Gems](https://guides.rubygems.org/what-is-a-gem/)

Reply:
I think @AMZN-stankowi may know